jekyll/_cci2/configuration-reference.md

```diff
@@ -361,12 +361,12 @@ jobs:
 
 #### **`branches`**
 
-Defines rules for allowing/blocking execution of some branches if Workflows are **not** configured and you are using 2.0 (not 2.1) config. If you are using [Workflows]({{ site.baseurl }}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows), job-level branches will be ignored and must be configured in the Workflows section of your `config.yml` file. If you are using 2.1 config, you will need to add a workflow in order to use filtering. See the [workflows](#workflows) section for details. The job-level `branch` key takes a map:
+Defines rules for allowing/blocking execution of jobs on specified branches if Workflows are **not** configured and you are using 2.0 (not 2.1) config. If you are using [Workflows]({{ site.baseurl }}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows), job-level branches will be ignored and must be configured in the Workflows section of your `config.yml` file. If you are using 2.1 config, you will need to add a workflow in order to use filtering. See the [workflows](#workflows) section for details. The job-level `branch` key takes a map:
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-only | N | List | List of branches that only will be executed
-ignore | N | List | List of branches to ignore
+only | N | String, List of Strings, Regex | Branch or list of branches for which the job will be executed
```

Comment:
For the type here, how do you feel about "String, Regex, or List of Strings and Regexes"? (Same for the below tables.) I feel that "String, List of Strings, Regex" is a bit ambiguous and could mean one of:

- a string
- a list of strings
- a regex

_or_, maybe:

- a string
- a list of string or regexes

But both of those are incomplete. We allow:

- a string
- a regex
- a list of strings or regexes (and you can freely mix strings and regexes)